### PR TITLE
activate preview A/B test by setting ONBOARDING_DOMAIN_TEST_DATE

### PR DESCRIPTION
**Pull request by biyeun**
@benrudolph

### CODE REVIEW

File: ansible/vars/production/production_public.yml [hunk 1]
```diff
@@ -204,6 +204,7 @@ localsettings:
   MAPS_LAYERS: "{{ localsettings_private.MAPS_LAYERS }}"
   MIA_THE_DEPLOY_BOT_API: "{{ localsettings_private.MIA_THE_DEPLOY_BOT_API }}"
   MOBILE_INTEGRATION_TEST_TOKEN: "{{ localsettings_private.MOBILE_INTEGRATION_TEST_TOKEN }}"
+  ONBOARDING_DOMAIN_TEST_DATE: (2016, 12, 8)
```
Comment (benrudolph) [inline ansible/vars/production/production_public.yml]: just checking that this will be a python string, not a tuple when it's put in the localsettings

Comment (biyeun) [inline ansible/vars/production/production_public.yml]: EXCELLENT CATCH THANK YOU. Updated.

**Comment by biyeun** [comment]
@benrudolph any idea if we're good to deploy this?

**Comment by benrudolph** [comment]
yep we're good :running_woman:

**Pull request merged**